Kernel/Graphics/Intel/DisplayConnectorGroup.cpp

```diff
@@ -277,7 +277,7 @@ StringView IntelDisplayConnectorGroup::convert_analog_output_register_to_string(
     }
 }
 
-void IntelDisplayConnectorGroup::write_to_general_register(RegisterOffset offset, u32 value) const
+void IntelDisplayConnectorGroup::write_to_general_register(RegisterOffset offset, u32 value)
```

Comment:
same comment for `Kernel/IntelGraphics: Mark write_to_general_register as non-const `